### DISCUSSION TITLE
Is it normal that before loading the page, instead of a number from the data, {{variable}} is displayed

### DISCUSSION THREAD

**Discussion by kr1ke** [vuejs/petite-vue#98] [Q&A] [answered]
Is it normal that before loading the page, instead of a number from the data, {{variable}} is displayed. Is this a bug or a norm? How to fix it. 

Example: 
```
import { createApp } from 'https://unpkg.com/petite-vue@0.2.2/dist/petite-vue.es.js'

createApp({variable: 0})

<div>{{count}}</div>
```

Before loading page in div displayed: {{count}}

**Comment by halivert** [comment, marked as answer]
You can use `v-cloak` directive to prevent that "flash"

https://v3.vuejs.org/api/directives.html#v-cloak

**Reply by kr1ke** [reply]
Thanks!

**Reply by ricky11** [reply]
It doesn't seem to work, placing v-cloak on the element, still renders the element, untill vue kicks in the javascript. (disable cache)

**Reply by halivert** [reply]
You should also use this code @ricky11 
```css
[v-cloak] {
  display: none !important;
}
```

**Reply by ricky11** [reply]
that did it. not mentioned in the docs about the sudden flash.
